interface/forms/gad7/new.php

```diff
@@ -0,0 +1,217 @@
+ <option selected value="undef" ><?php echo text($str_default); ?></option>
+    <option value="0"><?php echo text($str_not); ?></option>
+    <option value="1"><?php echo text($str_several); ?></option>
+    <option value="2"><?php echo text($str_more); ?></option>
+    <option value="3"><?php echo text($str_nearly); ?></option>
+    </select>
+    <br></br>
+</tr>
+ </table>
+<table>
+ <tr>
+<td>
+<span class="text"><?php echo xlt('Becoming easily annoyed or irritable'); ?></span>
+<select name="irritable_score" onchange="update_score(5, my_form.irritable_score.value);">
+ <option selected value="undef" ><?php echo text($str_default); ?></option>
+    <option value="0"><?php echo text($str_not); ?></option>
+    <option value="1"><?php echo text($str_several); ?></option>
+    <option value="2"><?php echo text($str_more); ?></option>
+    <option value="3"><?php echo text($str_nearly); ?></option>
+    </select>
+ <br></br>
+ </table>
+ </tr>
+<table>
+  <tr>
+<td>
+<span class="text"><?php echo xlt('Feeling afraid as if something awful might happen'); ?></span>
+<select name="fear_score" onchange="update_score(6, my_form.fear_score.value);">
+ <option selected value="undef" ><?php echo text($str_default); ?></option>
+    <option value="0"><?php echo text($str_not); ?></option>
+    <option value="1"><?php echo text($str_several); ?></option>
+    <option value="2"><?php echo text($str_more); ?></option>
+    <option value="3"><?php echo text($str_nearly); ?></option>
+    </select>
+ <br><br>
+</tr>
+ </table>
+<table  frame = above>
+<tr><td>
+ <span id="q8_place" class="text"><br></span>
+ </table>
+ <br></br>
+ <SCRIPT>
+function  check_all() {
+   // has each question been answered and save scores
+    var  flag=false;
+    var list='';
+    for (i=0; i<(no_qs-1); i++) { // last questionis optional
+          if ( !all_answered[i] ){
+          list = list+Number(i+1) + ',';
+          flag=true;
+          }
+    }
+    if (flag) {
+          list[list.length-1] = ' '; /* get rid of trailing comma */
+          alert(xl("Please answer all of the questions") + ": " + list + " " + xl("are unanswered"));
+           return false;
+     }
+    return true;
+  }
+  // warn if about to exit without saving answers - check that's what the user really wants
+function nosave_exit() {
+    var conf = confirm (<?php echo js_escape($str_nosave_confirm) ; ?>);
+
+    if (conf) {
+        window.location.href="<?php echo $GLOBALS['form_exit_url']; ?>";
+    }
+    return ( conf );
+}
+</script>
+<table frame=hsides><tr><td>
+ <span id="show_gad7_score"><b><?php echo xlt("Total GAD-7 score"); ?>:</b> </td>
+ </table>
+ <script>
+update_score("undef",gad7_score);
+ </script>
+ <br></br>
+ <table>
+ <tr><td>
+ <input type="Submit" value="<?php echo xla('Save Form'); ?>" style="color: #483D8B">
+ &nbsp &nbsp
+ <input type="button" value="<?php echo text($str_nosave_exit); ?>" onclick="top.restoreSession();return(nosave_exit());" style="color: #483D8B">
```

Comment:
attr() instead of text() since within html attributes:
```
<input type="button" value="<?php echo attr($str_nosave_exit); ?>" onclick="top.restoreSession();return(nosave_exit());" style="color: #483D8B">
```